bucket/sqlite.json

```
@@ -12,10 +13,14 @@
     ],
     "checkver": {
         "url": "https://www.sqlite.org/download.html",
-        "re": "sqlite-tools-win32-x86-(?<number>[\\d]+).zip[\\s\\S]+for SQLite version (?<version>[\\d.]+)\\."
+        "regex": "sqlite-tools-win32-x86-(?<number>[\\d]+).zip[\\s\\S]+for SQLite version (?<version>[\\d.]+)\\."
```

- `.*?` should work same as `[\\s\\S]+`
- named version group is not needed

```suggestion
        "regex": "sqlite-tools-win32-x86-(?<number>[\\d]+).zip.*?for SQLite version ([\\d.]+)\\."
```